mkosi/resources/man/mkosi.1.md

```diff
@@ -888,6 +888,10 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     in the ESP for compatibility with signed versions of grub which load
     the grub configuration from this location.
 
+    Note: When using `systemd-boot` or `systemd-boot-signed`, the image
```

Comment:
Since this is more of a footnote type thing and more paragraphs follow, could you move this at the end of the option description, i.e. after the paragraph that starts with "Kernels need to be placed"?